client/pchainclient.go

```diff
@@ -42,7 +68,9 @@ func NewPChainClient(ctx context.Context, endpoint string) PChainClient {
 
 	return pchainClient{
 		platformvmClient: platformvm.NewClient(endpoint),
+		indexerClient:    indexer.NewClient(endpoint + "/ext/index/P/block"),
```

Comment:
Should we allow to have a different endpoint for the indexer?

Reply:
Done